modules/preview-react/pill/lib/PillIcon.tsx

```diff
@@ -14,17 +14,24 @@ export interface PillIconProps extends Omit<SystemIconProps, 'icon'> {
    * @default `plusIcon`
    */
   icon?: CanvasSystemIcon;
+  /**
+   * The aria label for the icon
+   * @default 'add'
+   */
+  'aria-label'?: string;
 }
 
 export const PillIcon = createSubcomponent('span')({
   modelHook: usePillModel,
-})<PillIconProps>(({size, icon, ...elemProps}, Element) => {
+})<PillIconProps>(({size, icon, 'aria-label': ariaLabel = 'add', ...elemProps}, Element) => {
```

Comment:
```suggestion
})<PillIconProps>(({size, icon, ...elemProps}, Element) => {
```

Part of the other comment